test/command/test-base.rb

```diff
@@ -138,9 +138,9 @@ def test_select_command
       def test_brackets_format
         load = Groonga::Command::Base.new("load",
                                           :table => "Site",
+                                          :columns => "_key,title",
                                           :values => <<-VALUES)
   [
-  ["_key","title"],
```

Comment:
Could you add a test that doesn't use `:columns` and uses columns in `:values`.?